what,kind,bundleId,bundleVersion,description,time
started,bundle,org.eclipse.emf.common,2.5.0.v200906151043,"org.eclipse.emf.common",1323851271632
started,bundle,org.eclipse.emf.ecore,2.5.0.v200906151043,"org.eclipse.emf.ecore",1323851271730
started,bundle,org.eclipse.jem.util,2.0.201.v201001252130,"org.eclipse.jem.util",1323851271731
started,bundle,org.eclipse.wst.common.frameworks,1.1.300.v200904160730,"org.eclipse.wst.common.frameworks",1323851271741
started,bundle,org.eclipse.wst.common.emf,1.1.301.v200908181930,"org.eclipse.wst.common.emf",1323851271770
started,bundle,org.eclipse.emf.ecore.xmi,2.5.0.v200906151043,"org.eclipse.emf.ecore.xmi",1323851271840
started,bundle,org.eclipse.wst.common.emfworkbench.integration,1.1.301.v200908101600,"org.eclipse.wst.common.emfworkbench.integration",1323851271844
activated,workbench,org.eclipse.ui.workbench,3.5.2.M20100113-0800,"",1323851271903
started,bundle,org.eclipse.wst.common.modulecore,1.1.301.v201001252130,"org.eclipse.wst.common.modulecore",1323851271973
error,log,,,"Project facet jst.web has not been defined. It is used in plugin org.eclipse.wst.jsdt.web.core.",1323851272085
error,log,,,"Unable to find interpreter named: python 2.6. Available: python  (/hri/sit/builds/2011-08-16_17-00-00/External/python/2.6.6/lucid32/bin/python)",1323851274665
error,log,,,"Unable to find interpreter named: python 2.6. Available: python  (/hri/sit/builds/2011-08-16_17-00-00/External/python/2.6.6/lucid32/bin/python)",1323851274666
error,log,,,"Unable to find interpreter named: python 2.6. Available: python  (/hri/sit/builds/2011-08-16_17-00-00/External/python/2.6.6/lucid32/bin/python)",1323851274667
error,log,,,"Unable to resolve module for:L/DatasetGenerator/src/version.py",1323851274673
error,log,,,"Unable to find interpreter named: python 2.6. Available: python  (/hri/sit/builds/2011-08-16_17-00-00/External/python/2.6.6/lucid32/bin/python)",1323851281113
error,log,,,"Unable to find interpreter named: python 2.6. Available: python  (/hri/sit/builds/2011-08-16_17-00-00/External/python/2.6.6/lucid32/bin/python)",1323851281114
error,log,,,"Unable to find interpreter named: python 2.6. Available: python  (/hri/sit/builds/2011-08-16_17-00-00/External/python/2.6.6/lucid32/bin/python)",1323851281114
error,log,,,"Unable to resolve module for:L/DatasetGenerator/src/version.py",1323851281115
error,log,,,"Unable to find interpreter named: python 2.6. Available: python  (/hri/sit/builds/2011-08-16_17-00-00/External/python/2.6.6/lucid32/bin/python)",1323851290182
error,log,,,"Unable to find interpreter named: python 2.6. Available: python  (/hri/sit/builds/2011-08-16_17-00-00/External/python/2.6.6/lucid32/bin/python)",1323851290182
error,log,,,"Unable to find interpreter named: python 2.6. Available: python  (/hri/sit/builds/2011-08-16_17-00-00/External/python/2.6.6/lucid32/bin/python)",1323851290183
error,log,,,"Unable to resolve module for:L/DatasetGenerator/src/version.py",1323851290184
executed,command,org.eclipse.ui,3.5.2.M20100120-0800,"org.eclipse.ui.file.saveAll",1323851292151
deactivated,workbench,org.eclipse.ui.workbench,3.5.2.M20100113-0800,"",1323851292162
error,log,,,"Unable to find interpreter named: python 2.6. Available: python  (/hri/sit/builds/2011-08-16_17-00-00/External/python/2.6.6/lucid32/bin/python)",1323851292168
